docs/internals/contributing/writing-documentation.txt

```diff
@@ -241,7 +241,7 @@ __ http://sphinx-doc.org/markup/
 
         .. django-admin-option:: --traceback
 
-  To link, use ``:djadminopt:`--traceback```.
+  To link, use ``:option:`commandname --traceback```.
```

Comment:
Should we also mention that `commandname` can be omitted if were linking a global option?